### PR TITLE
Convert `standard/main` into ES6

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Converts `standard/main` to ES6. Why? I needed an easy one for the evening finish ...

## What is the value of this and can you measure success?

ES6!

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

No.

## Tested in CODE?

No.

### CODE REVIEW

File: static/src/javascripts/bootstraps/standard/main.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+                if (canViewContent) {
+                    const body = document.body;
+
+                    fastdom.write(() => {
+                        if (body) {
+                            body.classList.remove(requireClass);
+                        }
+                    });
+                } else {
```
Comment (regiskuckaertz) [inline static/src/javascripts/bootstraps/standard/main.js]: polyfill.io provides support for `Promise.prototype.finally`, so you can simplify a little here:
```
ajax(...)
.then(...)
.finally(redirect)
```

File: static/src/javascripts/bootstraps/standard/main.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                mediator.emitEvent('window:throttledScroll');
+                scrollRunning = false;
+            });
+        }
+    };
+
+    addEventListener(
+        window,
+        'scroll',
+        userPrefs.get('use-idle-callback') && 'requestIdleCallback' in window
```
Comment (regiskuckaertz) [inline static/src/javascripts/bootstraps/standard/main.js]: do we still want that? @sndrs

File: static/src/javascripts/bootstraps/standard/main.js [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+                const requiresPaidTier = !membershipAccess.includes(
+                    'paid-members-only'
+                );
+                // Check the users access matches the content
+                const canViewContent = requiresPaidTier
+                    ? !!resp.tier && resp.isPaidTier
+                    : !!resp.tier;
+
+                if (canViewContent) {
+                    const body = document.body;
```
Comment (SiAdcock) [inline static/src/javascripts/bootstraps/standard/main.js]: const { body } = document;

File: static/src/javascripts/bootstraps/standard/main.js [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+                    fastdom.write(() => {
+                        if (body) {
+                            body.classList.remove(requireClass);
+                        }
+                    });
+                } else {
+                    redirect();
+                }
+            })
+            .catch(() => redirect());
```
Comment (SiAdcock) [inline static/src/javascripts/bootstraps/standard/main.js]:
```js
.catch(redirect);
```

File: static/src/javascripts/bootstraps/standard/main.js (hunk 1 above)
Comment (gustavpursche) [inline static/src/javascripts/bootstraps/standard/main.js]: Interesting! `finally()` is at Stage 2 at the moment, so this shouldn't be in there. But yeah - if pf.io provides it, why noi/t?! 👍

**Review by SiAdcock — APPROVED** [review]
LGTM 👍

**Pull request merged**